pyEpiabm/pyEpiabm/intervention/travel_isolation.py

```diff
@@ -75,6 +75,11 @@ def __call__(self, time):
                                     selected_household = random.choice(
                                         existing_households)
                                     selected_household.add_person(person)
+                                    logging.info(f"Person {person.id} has "
+                                                 f"finished isolating and "
+                                                 f"has moved to household "
```

Comment:
Does this work? I would normally expect to these strings explicitly concatenated using an addition operator (+). Also you don't need an f-string for the sections that don't have any curly brackets - these can just be normal string, with no f before the quotation marks!

Reply:
The f strings have worked fine when I've tested them without explicit concatenation (they give the correct logging/error messages) and the f-string for sections without curly braces keep the format/alignment of the string the same. You can have multi-line f strings like this which concatenate without the plus operator and it works fine

Reply:
You learn something new everyday!